qubes/vm/appvm.py

```diff
@@ -77,7 +85,7 @@ def __init__(self, app, xml, **kwargs):
                     qubes.property.bool(None, None, node_dispvm_allowed.text)
                 node_dispvm_allowed.getparent().remove(node_dispvm_allowed)
 
-        self.volume_config = copy.deepcopy(self.default_volume_config)
+        self.volume_config = copy.deepcopy(AppVM.default_volume_config)
```

Comment:
This looks wrong - will break in a surprising way if there would be a class inheriting from AppVM that uses different default_volume_config.